api/AuthenticatorAttestationResponse.json

```diff
@@ -130,6 +130,57 @@
             "deprecated": false
           }
         }
+      },
+      "getTransports": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AuthenticatorAttestationResponse/getTransports",
+          "support": {
+            "chrome": {
+              "version_added": "75"
```

Comment:
There is no guarantee these will ship in 75. I'm speaking as Google, not as Joe Medley. Please change to `false`.